Casks/v/vcv-rack.rb

```diff
@@ -12,7 +15,20 @@
     regex(/###\s(\d+(?:\.\d+)+)/i)
   end
 
-  pkg "RackFree-#{version}-mac-x64.pkg"
+  pkg "RackFree-#{version}-mac-#{arch}.pkg"
 
   uninstall pkgutil: "com.vcvrack.rack"
+
+  zap trash: [
+    "~/Documents/Rack2/*.json",
+    "~/Documents/Rack2/autosave",
+    "~/Documents/Rack2/log.txt",
+    "~/Documents/Rack2/plugins-mac-arm64",
+    "~/Documents/Rack2/plugins-mac-x64",
+    "~/Library/Application Support/Rack2/*.json",
+    "~/Library/Application Support/Rack2/autosave",
+    "~/Library/Application Support/Rack2/log.txt",
+    "~/Library/Application Support/Rack2/plugins-mac-arm64",
+    "~/Library/Application Support/Rack2/plugins-mac-x64",
```

Comment:
This can be simplified down to directories.  All files inside will also be removed.

```suggestion
    "~/Documents/Rack2",
    "~/Library/Application Support/Rack2",
```

Reply:
This doesn't work as Patches the user makes are by default also stored in a subfolder called patches. And as far as I understood the zap requirements user created files shouldn't be removed.

Reply:
Correct.  Which was a follow-up question on including the `~/Documents/Rack2` folder in this to start with.  This is fine then, just a bit less than ideal.  But appreciate the work done here!

Reply:
Thank you :)